### ISSUE TITLE
[v3] explicitly need to use `styles` for a variant to be detected

### ISSUE THREAD

**Issue by jeremybarbet** [jpudysz/react-native-unistyles#456]
### Description

Hey,

Not sure if it's a bug report per se or an improvement. 

When using variants, the babel plugin will explicitly looks for a `styles` object to get the styles from (https://github.com/jpudysz/react-native-unistyles/blob/main/plugin/variants.js#L6)

I personally like to name my styles object `s` to keep it short, which means the following is not detected.

```tsx
const MyComponent = () => {
   s.useVariants(...)
}

const s = StyleSheet.create(...
```

Should we be able to defined a custom name through the `babel.config.js` or there is a smarter way to detect a style object with any custom name?

### Steps to Reproduce

1. Name your styles object `s`
2. Try to set `useVariants` 

### Unistyles Version

3.0.0-beta.4

### React Native Version

0.76.5

### Platforms

iOS

### Expo

Yes

**Comment by jpudysz** [comment]
Haha, I've never seen such renames, but for sure this should be supported 😄

**Comment by jeremybarbet** [comment]
Any direction on how it should be supported? I would be happy to contribute if it's through babel config

**Comment by jpudysz** [comment]
Yes, it's related to this one: https://github.com/jpudysz/react-native-unistyles/issues/436

I need to change detection strategy
